python/ray/_private/worker.py

```diff
@@ -2650,6 +2662,15 @@ def get(
         if isinstance(object_refs, CompiledDAGRef):
             return object_refs.get(timeout=timeout)
 
+        if isinstance(object_refs, list):
+            allCompiledDAGRefs = True
```

Comment:
```suggestion
            all_compiled_dag_refs = True
```
(we use underscore naming for python)